features/volgindicaties.feature

```diff
@@ -0,0 +1,154 @@
+    2. Wanneer geen enkele gebruiker een actieve volgindicatie op een buitengemeentelijke persoon, mag er geen actieve volgindicatie zijn voor deze persoon bij het GBA-V.
+    3. Voor een binnengemeentelijke persoon mag er geen actieve volgindicatie zijn in het GBA-V.
+
+   Volgindicaties in GBA-V kunnen worden gewijzigd n.a.v.:
+      - Een PUT /volgindicaties request
+      - Het bereiken van een datum
+      - Een binnengemeentelijke wijziging (de update applicatie ontvangt van een StUF npsLk01 kennisgeving uit het burgerzakensysteem)
+        - persoon is nu ingeschreven in deze gemeenten en applicatie ontvangt StUF mutatiesoort='V': uitverhuizing
+        - persoon is nu ingeschreven in andere gemeenten en applicatie ontvangt StUF mutatiesoort≠'V' (normaliter 'T'): inverhuizing
+ 
+
+   Scenario: Een gebruiker zet als eerste een volgindicatie op een binnengemeentelijk persoon
+      Gegeven de persoon met burgerservicenummer "999995066" is ingeschreven in deze gemeente
+      En er is geen enkele volgindicatie gezet voor de persoon met burgerservicenummer "999995066"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999995066" en geen einddatum
+      Dan stuurt de applicatie GEEN Ap01 bericht met burgerservicenummer "999995066" naar het GBA-V 
+
+   Scenario: Een gebruiker zet als eerste een volgindicatie op een buitengemeentelijk persoon
+      Gegeven de persoon met burgerservicenummer "999993653" is NIET ingeschreven in deze gemeente
+      En er is geen enkele volgindicatie gezet voor de persoon met burgerservicenummer "999993653"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999993653" en geen einddatum
+      Dan stuurt de applicatie een Ap01 bericht met burgerservicenummer "999993653" naar het GBA-V 
+
+   Scenario: Een gebruiker zet als eerste een volgindicatie op een persoon met einddatum in de toekomst
+      Gegeven de persoon met burgerservicenummer "999993483" is NIET ingeschreven in deze gemeente
+      En er is geen enkele volgindicatie gezet voor de persoon met burgerservicenummer "999993483"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999993483" met einddatum "2021-03-15"
+      Dan stuurt de applicatie een Ap01 bericht met burgerservicenummer "999993483" naar het GBA-V
+
+   Scenario: Een gebruiker zet een volgindicatie op een persoon die al gevolgd wordt door een andere gebruiker
+      Gegeven de persoon met burgerservicenummer "999990457" is NIET ingeschreven in deze gemeente
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999990457" en einddatum "2021-06-30"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999990457" en geen einddatum
+      Dan stuurt de applicatie GEEN Ap01 bericht met burgerservicenummer "999990457" naar het GBA-V
```

Comment:
Ik weet niet of Ap01 een einddatum heeft. Wat ik nu concludeer is dat ik als Update API geen Ap01 bericht stuur met einddatum gevuld. Want als de eerste volgindicatie voor een buitengemeentelijk persoon een einddatum heeft en een volgende volgindatie geen einddatum, dan moet een nieuwe Ap01 bericht worden gestuurd om de einddatum te wijzigen, denk ik

Reply:
een Ap01 heeft geen einddatum. De Av01 moet dus gestuurd worden op de dag van de laatste einddatum.